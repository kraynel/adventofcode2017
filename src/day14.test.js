const { countBits, countIslands } = require("./day14");
const fs = require("fs");

describe("countBits", () => {
  it("should count", () => {
    expect(countBits("flqrgnkx")).toBe(8108);
  });

  it("should get the value", () => {
    expect(countBits("ugkiagan")).toMatchSnapshot();
  });
});

describe("countIslands", () => {
  it("should islands", () => {
    expect(countIslands("flqrgnkx")).toBe(1242);
  });

  it("should get the value", () => {
    expect(countIslands("ugkiagan")).toMatchSnapshot();
  });
});
