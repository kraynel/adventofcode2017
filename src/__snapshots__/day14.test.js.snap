// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`countBits should get the value 1`] = `8292`;

exports[`countIslands should get the value 1`] = `1069`;
